### ISSUE TITLE
Python SDK No Longer Supported

### ISSUE THREAD

**Issue by kemra102** [aws-solutions/aws-media-services-application-mapper#182]
So we got a couple of emails today relating to 2 AWS accounts we have running MSAM and it seems that the Lambdas are using functionality that will no longer be supported starting 2021-04-1:

```
Hello,
We are contacting you regarding actions needed on your account(s) to prevent disruption to your CloudFormation stacks. Starting on April 01, 2021, AWS Lambda will no longer support the botocore.requests library in Python runtimes [1][2]. If you are using the cfn-response module for a Lambda-backed custom resource, you must modify your Lambda function resource's Code or Runtime property and update your stack(s) in order to get the latest version of the cfn-response module which removed its dependency on botocore.requests. If you do not update your Lambda function by April 01, 2021, you will no longer be able to create, update or delete your custom resource. If you are updating the Runtime property, please include the lambda:UpdateFunctionCode action in your IAM policy.

If you are hosting your Lambda function code from an S3 bucket, you can refer to the updated cfn-response module's source code in our documentation [3]. For more information on how to update lambda function can be found in knowledge center [4].

Please contact AWS Support if you have any questions or concerns [5].

[1] https://aws.amazon.com/blogs/compute/upcoming-changes-to-the-python-sdk-in-aws-lambda/
[2] https://aws.amazon.com/blogs/developer/removing-the-vendored-version-of-requests-from-botocore/
[3] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html
[4] https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-cfn-response-lambda/
[5] https://aws.amazon.com/support/

Sincerely,
Amazon Web Services
```

My understanding is that this mean MSAM will stop working on 2021-04-01. If so this will obviously need to be fixed ASAP so that we can all get the changes rolled out before that date.

**Comment by JimTharioAmazon** [comment]
Can you tell us the version or build ID of MSAM you have installed? We haven't used botocore.requests package in the code for some time. You may need to update the version of MSAM installed in your accounts to prevent disruption.

**Comment by kemra102** [comment]
Build ID: `1604080050` (MSAM v1.7.5)

**Comment by JimTharioAmazon** [comment]
After a fresh review of our change history, the team here is in agreement we addressed this in 1.5.0 when the botocore.requests deprecation notice was first sent in early 2020, and this is not a problem for MSAM 1.5.0 and later. 

We do not use the cfn-response module for our CloudFormation custom resources.

We believe these emails are being widely broadcast to anyone using Lambda, Python and CloudFormation together.
